src/libtools/edge.cc

```diff
@@ -30,37 +30,37 @@ namespace tools {
 using namespace std;
 
 Edge::Edge(int ID1, int ID2) {
-  vertices = minmax({ID1,ID2});
+  vertices_ = vector<int>{min({ID1,ID2}), max({ID1,ID2})};
 }
 
-int Edge::getOtherV(int ver) const {
-  if (ver == vertices.first) {
-    return vertices.second;
+int Edge::getOtherEndPoint(int ver) const {
```

Comment:
There are only ever two endpoints in an edge, the larger of the two is always placed as the second vertex. 

The method getOtherEndPoint takes either the first or second EndPoint and returns the other. GetEndPointSecond implies that the second vertex is always returned. getEndPointFirst would imply that the first vertex is always returned. 

Furthermore, the underlying data has been changed to a vector, and the reduced edge class can have more than two vertices stored in it (though it will still only ever have two endpoints). I would argue it is actually a very appropriate name, unless you can give me some more ideas.

Reply:
okay that makes sense, then keep the name